.travis.yml

```diff
@@ -5,8 +5,7 @@ sudo: required
 services:
 - docker
 
-python:
-  - "3.6"
+dist: xenial
```

Comment:
FYI, Github travis CI also has bionic distro available.

Reply:
Yup, was just trying to get it working on the version we were using in the past before moving it forward.